README.md

```diff
@@ -1,70 +1,214 @@
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- Connection between rails and React.
+- Connected to posgresql.
+- Authentication.
+- Users can reserve trips.
+- Users can see their own reservations.
+- Users can view available trips from the home page.
+- Users can see the details of each trip.
+- Users can add/delete trips.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### `npm run eject`
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+<!-- GETTING STARTED -->
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## 💻 Getting Started <a name="getting-started"></a>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+To get a local copy up and running, follow these steps.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+### Prerequisites
 
-## Learn More
+- Computer that has an IDE installed.
+- npm.
+- Git.
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### Front-end
+[front-end](https://github.com/Mhamad-Raad/final-capstone-frontend)
 
-To learn React, check out the [React documentation](https://reactjs.org/).
 
-### Code Splitting
+### Backend
+[back-end](https://github.com/Mhamad-Raad/final-capstone-backend)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
 
-### Analyzing the Bundle Size
+## 💻 Deployment <a name="triangular_flag_on_post-deployment"></a>
+In order to to get the production version of the app do the followings:
+- npm run build.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
 
-### Making a Progressive Web App
+## Live Version <a name="Live"></a>
+[Link](https://remarkable-buttercream-3be0d2.netlify.app/)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Setup
```

Comment:
_note: Since there is no backend PR for me to review, I'll include my comments here.  Basing review on the `dev` branch of the backend repo._

# Backend project comments

- [ ] This whole section of your backend readme is being rendered as code, that is why it's confusing, and the links don't show up:


![image](https://user-images.githubusercontent.com/57421823/224180324-3cbd55de-143a-43ba-b1c1-f77d0a6f3c13.png)

Kindly correct this bug in your markdown so the readme renders as intended. 👍 

- [ ]  You are using Rspec, but the "test" folder still exists in your backend project, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

- [ ] On your backend readme, you have included the command `rails db:migrate`, which is correct.  Please also include the required commands `rails db:create` and `rails db:seed` that your users will need to run the project.

- [ ] On your backend project, kindly include a "Tests" section with detailed instructions about how to run your tests:

```bash
rspec ./spec/models
rspec ./spec/controllers
```